README.md

```diff
@@ -70,17 +77,6 @@ In order to translate enum values for editor, define following in language XML f
 
 This is variation of `EnumEditorDescriptor` with one difference: by using `NullableEnumEditorDescriptor` you will get extra "empty" choice in the dropdown which will set null value to your property
```

Comment:
This is a variation of `EnumEditorDescriptor` with one difference: by using `NullableEnumEditorDescriptor` you will get an extra "empty" choice in the dropdown which will set a null value to your property